modules/scale-up-down-autoscaler-hcp.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-machine-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="scale-up-down-autoscaler-hcp_{context}"]
+= Scaling up and down workloads in a hosted cluster
+
+On {hcp}, you can scale up and down workloads in your hosted cluster by using the cluster autoscaler.
+
+.Prerequisites
+
+* You have created the `HostedCluster` and `NodePool` resources.
+
+.Procedure
+
+. Enable cluster autoscaling to set the scaling behavior to `ScaleUpAndScaleDown` for your hosted cluster. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> \
+  hostedcluster <hosted_cluster_name> \
+  --type=merge \
+  --patch='{"spec": {"autoscaling": {"scaling": "ScaleUpAndScaleDown", "maxPodGracePeriod": 60, "scaleDown": {"delayAfterAddSeconds": 60, "delayAfterDeleteSeconds": 60, "delayAfterFailureSeconds": 60, "unneededDurationSeconds": 60, "utilizationThresholdPercent": 50}}}}'
+----
+
+. Remove the `spec.replicas` field from the `NodePool` resource so that cluster autoscaler can manage the node count. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> \
+  nodepool <node_pool_name> \
+  --type=json  \
+  --patch='[{"op": "remove", "path": "/spec/replicas"}]'
+----
+
+. Enable autoscaling to configure the minimum and maximum node counts for your node pools. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> \
+  nodepool <nodepool_name> \
+  --type=merge --patch='{"spec": {"autoScaling": {"max": 3, "min": 1}}}'
+----
+
+.Verification
+
+. Verify that all worker nodes are in the `Ready` status by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <hosted_cluster_name>.kubeconfig get nodes
+----
+
+. Verify that the cluster autoscaler pod contains the `--expander=priority,least-waste` argument by running the following command:
++
+[source,terminal]
+----
+$ oc get pod -n <hosted_control_plane_namespace> \
+  <cluster_autoscaler_pod_name> -o jsonpath='{.spec.containers[0].args}'
+----
```

Comment:
Shall we add about the nodes will scale up  with workloads, and will scale down after workload gets deleted ?

Reply:
Addressed